src/seed/seedCategories.js

```
@@ -0,0 +1,61 @@
+const Category = require('~/models/category');
+const logger = require('~/logger/logger');
+
+const SeedCategories = {
+  createCategories: async () => {
+    try {
+      const categories = [
+        {
+          name: 'Design',
+          appearance: {
+            icon: 'icons/design.svg',
+            color: '#FF5733'
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)